<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-9DH3JAO" failures="20" tests="50" name="Automation Test cases" time="468.139" errors="0" timestamp="2023-02-16T02:35:30 IST">
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_Mandatory_Field" time="3.79"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_BY_Providing_All_Fileds" time="3.777"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Notification_BY_NotProviding_All_Fileds_BySubmiting" time="1.863"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_Newsletter_NO" time="3.822"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_Newsletter_YES" time="3.575"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Different_Pass_con_pass" time="3.519"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Existing_Account" time="3.886"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Invalid_Email" time="2.652"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Invalid_Email" time="2.006"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Invalid_Email" time="2.286"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Invalid_Phone" time="2.567">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialsninja.qa.testCases.TestRegister.verify_Registering_By_Invalid_Phone(TestRegister.java:192)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_Registering_By_Invalid_Phone -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_By_Invalid_Phone" time="2.999">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.tutorialsninja.qa.testCases.TestRegister.verify_Registering_By_Invalid_Phone(TestRegister.java:192)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- verify_Registering_By_Invalid_Phone -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_With_Space_Field" time="2.657">
    <failure type="java.lang.AssertionError" message="expected [Telephone must be between 3 and 32 characters!] but found [Password]">
      <![CDATA[java.lang.AssertionError: expected [Telephone must be between 3 and 32 characters!] but found [Password]
at com.tutorialsninja.qa.testCases.TestRegister.verify_Registering_With_Space_Field(TestRegister.java:222)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_Registering_With_Space_Field -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_Complexity_Password" time="12.664">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;//input[@id=&amp;apos;input-password&amp;apos;]/following::div[1]&amp;quot;}
  (Session info: MicrosoftEdge=110.0.1587.41)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &amp;apos;4.6.0&amp;apos;, revision: &amp;apos;79f1c02ae20&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.5&amp;apos;
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [1c452d371543aec88935f1aad1256c07, findElement {using=xpath, value=//input[@id=&amp;apos;input-password&amp;apos;]/following::div[1]}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 110.0.1587.41, ms:edgeOptions: {debuggerAddress: localhost:56271}, msedge: {msedgedriverVersion: 110.0.1587.41 (dec2c0803210..., userDataDir: C:\Users\Mahi\AppData\Local...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56271/devtoo..., se:cdpVersion: 110.0.1587.41, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1c452d371543aec88935f1aad1256c07">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@id='input-password']/following::div[1]"}
  (Session info: MicrosoftEdge=110.0.1587.41)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.6.0', revision: '79f1c02ae20'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [1c452d371543aec88935f1aad1256c07, findElement {using=xpath, value=//input[@id='input-password']/following::div[1]}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 110.0.1587.41, ms:edgeOptions: {debuggerAddress: localhost:56271}, msedge: {msedgedriverVersion: 110.0.1587.41 (dec2c0803210..., userDataDir: C:\Users\Mahi\AppData\Local...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56271/devtoo..., se:cdpVersion: 110.0.1587.41, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 1c452d371543aec88935f1aad1256c07
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:365)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:357)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy15.getText(Unknown Source)
at com.tutorialsninja.qa.pageObjects.RegisterPageObjects.password_actual_WarningMessage(RegisterPageObjects.java:88)
at com.tutorialsninja.qa.testCases.TestRegister.verify_Registering_Complexity_Password(TestRegister.java:243)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- verify_Registering_Complexity_Password -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_Spaces_Trimmed" time="2.967">
    <failure type="java.lang.AssertionError" message="did not expect [    test    ] but found [    test    ]">
      <![CDATA[java.lang.AssertionError: did not expect [    test    ] but found [    test    ]
at com.tutorialsninja.qa.testCases.TestRegister.verify_Registering_Spaces_Trimmed(TestRegister.java:261)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- verify_Registering_Spaces_Trimmed -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_PrivacyPolicy_isSelected" time="0.931"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_PrivacyPolicy_Without_Selecting" time="7.406"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestRegister" name="verify_Registering_enterPassword_WithoutConfirmPassword" time="3.083"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_With_Valid_Credential" time="2.547"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_With_InValid_Credential" time="2.119"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_With_Valid_password_invalid_userName" time="2.833"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_With_Valid_userName_invalid_password" time="2.742"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_Without_Credentials" time="2.164"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_Forgetten_Password" time="1.178"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_With_Valid_Credential_BrowserBack" time="11.776">
    <failure type="org.openqa.selenium.NoSuchElementException" message="no such element: Unable to locate element: {&amp;quot;method&amp;quot;:&amp;quot;xpath&amp;quot;,&amp;quot;selector&amp;quot;:&amp;quot;(//div[@class=&amp;apos;well&amp;apos;])[2]/h2&amp;quot;}
  (Session info: MicrosoftEdge=110.0.1587.41)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &amp;apos;4.6.0&amp;apos;, revision: &amp;apos;79f1c02ae20&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.5&amp;apos;
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [f5c596ed2c0c6eb253cba202096dabea, findElement {using=xpath, value=(//div[@class=&amp;apos;well&amp;apos;])[2]/h2}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 110.0.1587.41, ms:edgeOptions: {debuggerAddress: localhost:56803}, msedge: {msedgedriverVersion: 110.0.1587.41 (dec2c0803210..., userDataDir: C:\Users\Mahi\AppData\Local...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56803/devtoo..., se:cdpVersion: 110.0.1587.41, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f5c596ed2c0c6eb253cba202096dabea">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"(//div[@class='well'])[2]/h2"}
  (Session info: MicrosoftEdge=110.0.1587.41)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.6.0', revision: '79f1c02ae20'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.5'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [f5c596ed2c0c6eb253cba202096dabea, findElement {using=xpath, value=(//div[@class='well'])[2]/h2}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 110.0.1587.41, ms:edgeOptions: {debuggerAddress: localhost:56803}, msedge: {msedgedriverVersion: 110.0.1587.41 (dec2c0803210..., userDataDir: C:\Users\Mahi\AppData\Local...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56803/devtoo..., se:cdpVersion: 110.0.1587.41, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f5c596ed2c0c6eb253cba202096dabea
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:365)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:357)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy15.getText(Unknown Source)
at com.tutorialsninja.qa.pageObjects.LoginPageObjects.returningCustomer_actual_text(LoginPageObjects.java:105)
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_With_Valid_Credential_BrowserBack(TestLogin.java:114)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_With_Valid_Credential_BrowserBack -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_afterLogin_browserBack" time="2.524">
    <failure type="java.lang.AssertionError" message="Account Logout expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Account Logout expected [true] but found [false]
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_afterLogin_browserBack(TestLogin.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_afterLogin_browserBack -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="2.6">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.467">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.628">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.517">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.511">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.625">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.608">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.541">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="2.586">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_MultipleTime_Unsuccessful_attempt" time="1.591">
    <failure type="java.lang.AssertionError" message="expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Warning: Your account has exceeded allowed number of login attempts. Please try again in 1 hour.] but found []
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_MultipleTime_Unsuccessful_attempt(TestLogin.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_MultipleTime_Unsuccessful_attempt -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_Copying_TEXT_PasswordFiled" time="1.295"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_Code_In_SourceCode_PasswordFiled" time="1.785">
    <failure type="java.lang.AssertionError" message="did not expect [xyzabc123   ] but found [xyzabc123   ]">
      <![CDATA[java.lang.AssertionError: did not expect [xyzabc123   ] but found [xyzabc123   ]
at com.tutorialsninja.qa.testCases.TestLogin.verify_login_Code_In_SourceCode_PasswordFiled(TestLogin.java:174)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- verify_login_Code_In_SourceCode_PasswordFiled -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_after_Changing_Password" time="5.18"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_able_to_navigate_to_otherPages" time="2.843"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogin" name="verify_login_after_Changing_Password_and_Changing_toOriginal_password" time="5.678"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_By_LoggingOut_From_MyAccount_Dropdown" time="2.933"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_By_LoggingOut_From_LogoutOption_OnPage" time="2.537"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_By_LoggingOut_And_BrowsingBack" time="2.529">
    <failure type="java.lang.AssertionError" message="expected [Account Logout] but found [My Account]">
      <![CDATA[java.lang.AssertionError: expected [Account Logout] but found [My Account]
at com.tutorialsninja.qa.testCases.TestLogout.verify_Logout_By_LoggingOut_And_BrowsingBack(TestLogout.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- verify_Logout_By_LoggingOut_And_BrowsingBack -->
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_Option_NotDisplayed_BeforeLogin_Account_Drodown" time="10.204"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_Option_NotDisplayed_BeforeLogin_UnderRight_Column" time="10.035"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestLogout" name="verify_Logout_AfterLogout_FromOneAccount_AnotherAccount" time="2.604"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestForgetPassword" name="verify_forgetPassword_Withoutout_reset_but_IntitiateProcess" time="2.003"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestForgetPassword" name="verify_forgetPassword_With_nonRegistered_Account" time="1.243"/>
  <testcase classname="com.tutorialsninja.qa.testCases.TestForgetPassword" name="verify_forgetPassword_Without_providing_EmailAddress" time="1.162">
    <failure type="java.lang.AssertionError" message="E-Mail must be between 4 and 20 characters!&amp;apos; should be displayed for &amp;apos;E-Mail Address&amp;apos; field expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: E-Mail must be between 4 and 20 characters!' should be displayed for 'E-Mail Address' field expected [true] but found [false]
at com.tutorialsninja.qa.testCases.TestForgetPassword.verify_forgetPassword_Without_providing_EmailAddress(TestForgetPassword.java:69)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- verify_forgetPassword_Without_providing_EmailAddress -->
</testsuite> <!-- Automation Test cases -->
